src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -468,32 +519,12 @@ var Marathon = React.createClass({
             </ul>
           </div>
         </nav>
-        <div className="container-fluid">
-          <TogglableTabsComponent activeTabId={this.state.activeTabId} >
-            <TabPaneComponent id="apps">
-              <button type="button" className="btn btn-success navbar-btn"
-                  onClick={this.showNewAppModal} >
-                + New App
-              </button>
-              <AppListComponent
-                collection={this.state.collection}
-                onSelectApp={this.showAppModal}
-                fetchState={this.state.fetchState}
-                ref="appList" />
-            </TabPaneComponent>
-            <TabPaneComponent
-                id="deployments"
-                onActivate={this.props.fetchAppVersions} >
-              <DeploymentsListComponent
-                deployments={this.state.deployments}
-                destroyDeployment={this.destroyDeployment}
-                fetchState={this.state.deploymentsFetchState} />
-            </TabPaneComponent>
-          </TogglableTabsComponent>
-        </div>
+        {page}
```

Comment:
Please wrap this in the `.container-fluid`, and get rid of that `.toggleabletabs-container` container. Before creating new styles please check that Bootstrap doesn't already implement functionality that you can use